cloud-sql/postgres/ConnectUnix.cs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// [START cloud_sql_postgres_dotnet_ado_connect_unix]
+using Npgsql;
+using System;
+
+namespace CloudSql
+{
+    public class PostgreSqlUnix
+    {
+        public static NpgsqlConnectionStringBuilder NewPostgreSqlUnixSocketConnectionString()
+        {
+            // Equivalent connection string:
+            // "Server=<dbSocketDir>/<INSTANCE_CONNECTION_NAME>;Uid=<DB_USER>;Pwd=<DB_PASS>;Database=<DB_NAME>"
+            String dbSocketDir = Environment.GetEnvironmentVariable("INSTANCE_UNIX_SOCKET") ?? "/cloudsql";
```

Comment:
Same here: Let's remove the `/cloudsql` alternative. `INSTANCE_UNIX_SOCKET` should be the full path.

Reply:
Done.